VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -111,6 +121,11 @@ public void onPageLoaded(List<T> newItems, boolean hasMore) {
                     hasMoreItems = hasMore;
                     loading = false;
                 }
+
+                private boolean isResetCalled() {
```

Comment:
This method can be renamed to `isAbandoned()` for better accuracy, since calling `reset()` on the controller causes all the current loaders (and their callbacks) to be abandoned.